docs/userguide.rst

```diff
@@ -37,86 +39,10 @@ You can install all necessary dependencies as follows:
 .. code-block:: console
 
    $ brew install kubernetes-cli
-   $ brew install hyperkit
    $ brew cask install minikube
+   $ brew cask install virtualbox
    $ brew install kubernetes-helm
-   $ brew install docker-machine-driver-hyperkit
-
-Ubuntu
```

Comment:
:+1: 

The detailed recipes for various operating systems can live as [forum.reana.io](http://forum.reana.io) posts.